perma_web/static/bundles/create.js

```diff
@@ -3400,6 +3400,10 @@ webpackJsonp([1],[
 	      // update display title when saved
 	      textarea.closest('.item-container').find('.item-title span').text(data.title);
 	    });
+	  }).on('input', '.link-description', function () {
+	    var textarea = $(this);
+	    var guid = getLinkIDForFormElement(textarea);
+	    LinkHelpers.saveInput(guid, textarea, textarea.prevAll('.description-save-status'), 'description');
 	  })
```

Comment:
Do you know why we are using prevAll here, and above? It seems to me we only want a single element? (Will merge anyone, this is clearly idiomatic to this file.